src/zenml/models/v2/core/run_metadata.py

```diff
@@ -30,14 +30,12 @@
 class RunMetadataRequest(WorkspaceScopedRequest):
     """Request model for run metadata."""
 
-    resource_id: UUID = Field(
-        title="The ID of the resource that this metadata belongs to.",
-    )
-    resource_type: MetadataResourceTypes = Field(
-        title="The type of the resource that this metadata belongs to.",
+    resources: List[Tuple[UUID, MetadataResourceTypes]] = Field(
```

Comment:
Same as above

Reply:
Closing this one, will follow the discussion above.